cluster-autoscaler/simulator/cluster.go

```diff
@@ -156,7 +156,7 @@ func (r *RemovalSimulator) SimulateNodeRemoval(
 	if err != nil {
 		klog.Errorf("Can't retrieve node %s from snapshot, err: %v", nodeName, err)
 	}
-	klog.V(2).Infof("%s for removal", nodeName)
+	klog.V(2).Infof("Simulating node %s for removal", nodeName)
```

Comment:
nit: I'd drop `for`: `Simulating node %s removal` seems to reflect exactly what's going on.

Reply:
Thank you @x13n.  That's more concise - I agree.  (Change pushed.)